docs/generate-time-series/go.rst

```diff
@@ -339,20 +357,30 @@ Then in the script's ``main`` function, create an infinite loop that gets the
                     fmt.Println("INSERT OK")
                 }
             }
-            fmt.Println("Sleeping for 5 seconds...")
-            time.Tick(time.Second * 5)
+            fmt.Println("Sleeping for 10 seconds...")
```

Comment:
@infoverload 10 seconds for consistency with the other tutorials, so the mock data has the same resolution no matter which tutorial you follow. this resolution (10 seconds) is mentioned explicitly in the next tutorial that shows the reader how to normalize the data